server/memphis_helper.go

```diff
@@ -382,6 +377,23 @@ func (s *Server) QueueSubscribe(subj, queueGroupName string, cb func(string, []b
 	return err
 }
 
+func (s *Server) subscribeOnGlobalAcc(subj, sid string, cb func(string, string, []byte)) error {
```

Comment:
sid == queue group ? if it does let's change it

Reply:
subscription id, following NATS' naming